test/test_prompt_generators.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import numpy as np
+
+from skimage.data import binary_blobs
+from skimage.measure import label
+
+
+class TestPromptGenerators(unittest.TestCase):
+
+    def _get_test_data(self):
+        data = binary_blobs(length=256)
+        labels = label(data)
+        return labels
+
+    def test_point_prompt_generator(self):
+        from micro_sam.prompt_generators import PointPromptGenerator
+        from micro_sam.util import get_cell_center_coordinates
+
+        labels = self._get_test_data()
+        label_ids = np.unique(labels)
+
+        mode = "v"  # this gets center points using vigra - regionprops throws error for asymmetrical blobs
+        centers, boxes = get_cell_center_coordinates(labels, mode=mode)
+        if mode == "v":
+            centers.pop(0)   # to avoid the background element
```

Comment:
This should be handled in `get_cell_center_coordinates` instead of the test (or other code that uses this).
The format of the output should be the same for all modes, so that it can be used interchangeably.

Reply:
And we should also set the default mode to `v` since it gives more correct results.

Reply:
Yes, that keeps the functionality alike. Updated it now, thanks!